allensdk/internal/api/mtrain_api.py

```diff
@@ -6,12 +6,15 @@
 import json
 import uuid
 
-from . import PostgresQueryMixin
-from allensdk.brain_observatory.behavior.session_apis.data_io import (
-    BehaviorLimsApi)
+from . import PostgresQueryMixin, db_connection_creator
 from allensdk.brain_observatory.behavior.trials_processing import EDF_COLUMNS
-from allensdk.core.auth_config import MTRAIN_DB_CREDENTIAL_MAP
+from allensdk.core.auth_config import MTRAIN_DB_CREDENTIAL_MAP, \
+    LIMS_DB_CREDENTIAL_MAP
 from allensdk.core.authentication import credential_injector
+from ...brain_observatory.behavior.data_objects import BehaviorSessionId
+from ...brain_observatory.behavior.data_objects.metadata.behavior_metadata\
```

Comment:
This is just my personal preference, but please promote these to absolute path imports, i.e.

`from allensdk.brain_observatory.behavior.data_objects.meta_data.... import`

It will make it easier to find the imported functions/classes whenever we have to maintain this file.